packages/styles/scss/themes/expressive/components/_code-snippet.scss

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/layout/scss/convert';
+
+//-----------------------------
+// Code snippet (expressive)
+//-----------------------------
+
+/// Code snippet styles (expressive)
+/// @access private
+/// @group code-snippet-expressive
+@mixin code-snippet-expressive {
+  // Single Line Snippet
+  .#{$prefix}--snippet--single {
+    height: $carbon--spacing-09;
+  }
+  .#{$prefix}--snippet--single .#{$prefix}--snippet-container {
+    height: $carbon--spacing-09;
+    padding: 0;
+  }
+  .#{$prefix}--snippet-button {
+    height: 3rem;
+    width: 3rem;
+  }
+  //Single Line Snippet Copy Button
+  .#{$prefix}--snippet__icon {
+    height: rem(20px);
+    width: rem(20px);
+  }
+  // Multi Line Snippet Copy Button
+  .#{$prefix}--snippet--multi .#{$prefix}--snippet-button {
+    height: $carbon--spacing-08;
```

Comment:
@oliviaflory did you want only the hover state of the copy button to be 48px by 48px? or did you want the button to always be 48px regardless of what state it is in?

Reply:
Also looking at the carbon code, it is using the `<CopyButton />` component so @mkothur we should probably move those changes to the `<CopyButton />` button instead like mentioned in the ticket.

Reply:
you mean to create another component for _copy-button.scss to have them in this file or change the file name _code-snippet.scss to _copy-button.scss?

Reply:
I mean create another file `_copy-button.scss` for the CopyButton component yes